cassandra-executor/src/main/java/io/mesosphere/mesos/frameworks/cassandra/CassandraExecutor.java

```diff
@@ -358,8 +409,34 @@ private CassandraNodeHealthCheckDetails performHealthCheck(@NotNull final Cassan
         return builder.build();
     }
 
-    private static CassandraNodeInfo buildInfo(JmxConnect jmx) throws UnknownHostException {
-        Nodetool nodetool = new Nodetool(jmx);
+    private void processAlive(ExecutorDriver driver) {
```

Comment:
I find the name of this method confusing. Is it supposed to make sure that the process is alive and prevent code from running that requires the process to be alive?  The way the method it calls looks like an assertion to me.